bin/validate.sh

```diff
@@ -3,6 +3,6 @@ set -euxo pipefail
 
 # TODO: Enable the valid templates
 # TODO: Switch to way-faster Python script
-find tests/translator/input -type f -name '*.yaml' | grep -v error_ | grep -v unsupported_resources | grep -v resource_with_invalid_type | shuf | while read -r template; do
+find tests/translator/input -type f -name '*.yaml' | grep -v error_ | grep -v null_events | grep -v unsupported_resources | grep -v resource_with_invalid_type |  while read -r template; do
```

Comment:
Was `null_events` supposed to skip two tests? If yes, clearer:

```suggestion
find tests/translator/input -type f -name '*.yaml' | grep -v error_ | grep -v function_with_null_events | grep -v state_machine_with_null_events | grep -v unsupported_resources | grep -v resource_with_invalid_type |  while read -r template; do
```

Reply:
Yes, it's supposed to skip two tests.